examples/marc/bin_marc_unicode.mrc

```diff
@@ -1 +1 @@
-01266cam  2200361 i 4500001001200000003000700012005001700019007000200036008004100038020003000079035002100109035002700130040002400157049003500181084001700216084001700233100011700250240002800367245006500395250002500460260004800485300001100533500007100544650004000615650003500655650004100690689003300731689003900764689001100803689003800814689001100852999004100863BV000023623DE-60420100617        t870612s1982             |||| 00||| und d  a3434004246$93-434-00424-6  a(OCoLC)157011841  a(DE-599)BVBBV000023623  aDE-604bgererakddb  aDE-12aDE-M347aDE-703aDE-525  aCU 2567$2rvk  aMS 1290$2rvk1 aƀƁƂƃƄƅƆƇƈƉƊƋƌƍƎƏƐƑƒƓƔƕƖƘƙƚƛƜƝƞƟƠơd1900-1980eVerfasser$0(DE-588)118536389$4aut10a?The? sane societyldt.10aΛΦϞЌЍЖ⁁⅀∰	ﬡﬢ－中英字典こんにちはß  a11., überarb. Aufl.  aFrankfurt am MainbEurop. Verl.-Anst.c1982  a354 S.  aFrüher u.d.T.: Fromm, Erich: Der moderne Mensch und seine Zukunft070(DE-588)4155776-1aFuturologie$2gnd070(DE-588)4038639-9aMensch$2gnd070(DE-588)4020588-5aGesellschaft$2gnd000(DE-588)4038639-9DsaMensch010(DE-588)4020588-5DsaGesellschaft0 5DE-604100(DE-588)4155776-1DsaFuturologie1 5DE-604  aoai:aleph.bib-bvb.de:BVB01-000000092
\ No newline at end of file
+01266cam  2200361 i 4500001001200000003000700012005001700019007000200036008004100038020003000079035002100109035002700130040002400157049003500181084001700216084001700233100011700250240002800367245006500395250002500460260004800485300001100533500007100544650004000615650003500655650004100690689003300731689003900764689001100803689003800814689001100852999004100863BV000023623DE-60420100617        t870612s1982             |||| 00||| und d  a343400424693-434-00424-6  a(OCoLC)157011841  a(DE-599)BVBBV000023623  aDE-604bgererakddb  aDE-12aDE-M347aDE-703aDE-525  aCU 25672rvk  aMS 12902rvk1 aƀƁƂƃƄƅƆƇƈƉƊƋƌƍƎƏƐƑƒƓƔƕƖƘƙƚƛƜƝƞƟƠơd1900-1980eVerfasser0(DE-588)1185363894aut10a?The? sane societyldt.10aΛΦϞЌЍЖ⁁⅀∰	ﬡﬢ－中英字典こんにちはß  a11., überarb. Aufl.  aFrankfurt am MainbEurop. Verl.-Anst.c1982  a354 S.  aFrüher u.d.T.: Fromm, Erich: Der moderne Mensch und seine Zukunft070(DE-588)4155776-1aFuturologie2gnd070(DE-588)4038639-9aMensch2gnd070(DE-588)4020588-5aGesellschaft2gnd000(DE-588)4038639-9DsaMensch010(DE-588)4020588-5DsaGesellschaft0 5DE-604100(DE-588)4155776-1DsaFuturologie1 5DE-604  aoai:aleph.bib-bvb.de:BVB01-000000092
```

Comment:
👍 from me on the changes to these two .mrc files.

 I'm not sure where these binary MARCs came from, I couldn't find them on IA, but those subfields look like they were missed out and treated as part of the 'e' subfield. You can see the included XML example is also incorrectly groups the subfields within 'e': https://github.com/internetarchive/openlibrary-client/blob/master/examples/marc/marc.xml#L39

I played around with yaz-marcdump and could produce 
``` 
<datafield tag="100" ind1="1" ind2=" ">
    <subfield code="a">Fromm, Erich</subfield>
    <subfield code="d">1900-1980</subfield>
    <subfield code="e">Verfasser</subfield>
    <subfield code="0">(DE-588)118536389</subfield>
    <subfield code="4">aut</subfield>
  </datafield>
```
  from the corrected binary marc above.

@mekarpeles , where did these records come from? Just worrying that there may a systematic source of these conflated subfields?